### ISSUE TITLE
iPad 7 (A10) on iOS 13.6 shuts itself down after jailbreaking with odyssey 1.2 and then reboots after 2 minutes blackscreen into nonjailbroken state.

### ISSUE THREAD

**Issue by Barry1601** [TheOdysseyJB/Odyssey#28]
So I previously was jailbroken with checkra1n  and I wanted to try the new odyssey release. I rootfsed first, then jailbroke after a good amount of tries. Then I installed libhooker, RocketBootstrap, PreferenceLoader. I installed some tweaks like dra1n. After that I left my iPad to charge, though 2-3 minutes later the iPad had a blackscreen and I couldnt turn it back on (even with hard-reboot), another minute and it restarted into non-jailbroken state. I tried to rejailbreak but its gets stuck. I tried rebooting countless times too and the succesrates are low and all the times the same issue with the blackscreen and restarting persits. I also ran succesion on my iPad, as I thought it may be something with me being jailbroken previously with checkra1n but the same issues. One of my friends has the same iPad (A10), running iOS 13.6.1, experiencing the same issues as me.  I hope this is like a bug and will get fixed soon, any help or infos would be appreciated!

**Comment by trungking** [comment]
Same problem here
Had to revert back to 1.1.2

**Comment by MrDandE** [comment]
Same problem with Iphone 7. Phone randomly shuts down but it does not turn back on even with hard reboot; it takes a couple of minutes to get it to boot. Jailbreak success rates pretty low; it often takes me 10 tries for it to work. And for some reason, there is a huge delay in rebooting my phone when I have odyssey. I switched from unc0ver to odyssey for a better battery life but so far its a hassle.

**Comment by Roonil** [comment]
I have an iPad 7th gen (A10) on iOS 13.6.1 and can confirm the above bug. After jailbreaking, whenever the device gets locked (eg. after pressing the power button once or letting the screen timeout do its job) and then waiting for about 10 seconds, I’m unable to turn the screen on, and then it reboots itself after 2-3 minutes. It’s quite annoying. Both 1.2.1 and 1.2.2 have this bug. Please look into this ASAP.

**Comment by tommysllee** [comment]
Same here. Ipad 6, 13.6.1, Same problem with Roonil...



> I have an iPad 7th gen (A10) on iOS 13.6.1 and can confirm the above bug. After jailbreaking, whenever the device gets locked (eg. after pressing the power button once or letting the screen timeout do its job) and then waiting for about 10 seconds, I’m unable to turn the screen on, and then it reboots itself after 2-3 minutes. It’s quite annoying. Both 1.2.1 and 1.2.2 have this bug. Please look into this ASAP.

Sir, only 1.2.1 and 1.2.2? So what if we use earlier version? Is it force-reboot-free?

**Comment by Roonil** [comment]
> Same here. Ipad 6, 13.6.1, Same problem with Roonil... But with 1.2 (downloaded nov 21st 2020)... I think i have to try 1.2.1 or 1.2.2, before trying 1.1.2...
> 
> > I have an iPad 7th gen (A10) on iOS 13.6.1 and can confirm the above bug. After jailbreaking, whenever the device gets locked (eg. after pressing the power button once or letting the screen timeout do its job) and then waiting for about 10 seconds, I’m unable to turn the screen on, and then it reboots itself after 2-3 minutes. It’s quite annoying. Both 1.2.1 and 1.2.2 have this bug. Please look into this ASAP.
> 
> Sir, only 1.2.1 and 1.2.2? So what if we use earlier version? Is it force-reboot-free?

Those versions unfortunately don’t support iOS 13.6.1 :( I had tried using 1.1.2 as mentioned above but it says unsupported on the jailbreak screen.

**Comment by tommysllee** [comment]
> > Same here. Ipad 6, 13.6.1, Same problem with Roonil... But with 1.2 (downloaded nov 21st 2020)... I think i have to try 1.2.1 or 1.2.2, before trying 1.1.2...
> > > I have an iPad 7th gen (A10) on iOS 13.6.1 and can confirm the above bug. After jailbreaking, whenever the device gets locked (eg. after pressing the power button once or letting the screen timeout do its job) and then waiting for about 10 seconds, I’m unable to turn the screen on, and then it reboots itself after 2-3 minutes. It’s quite annoying. Both 1.2.1 and 1.2.2 have this bug. Please look into this ASAP.
> > 
> > 
> > Sir, only 1.2.1 and 1.2.2? So what if we use earlier version? Is it force-reboot-free?
> 
> Those versions unfortunately don’t support iOS 13.6.1 :( I had tried using 1.1.2 as mentioned above but it says unsupported on the jailbreak screen.

You are right... I also tried just now... And i found that 1.2, 1.2.1, 1.2.2 are having same prob of this force restart..

**Comment by CallMeEchoCodes** [comment]
same issue on a ipad 6th gen running ios 13.2.2

**Comment by CrankySupertoon** [comment]
Same issue on an iPad 5 running ios13.5.1

**Comment by tommysllee** [comment]
Hi all, i get the solution

**Make your screen never locks automatically, AND Kill all apps before locking your screen**
